container/innerwindow.go

```diff
@@ -25,15 +24,15 @@ type InnerWindow struct {
 	Icon                                                fyne.Resource
 
 	title   string
-	content fyne.CanvasObject
+	content *fyne.Container
 }
 
 // NewInnerWindow creates a new window border around the given `content`, displaying the `title` along the top.
 // This will behave like a normal contain and will probably want to be added to a `MultipleWindows` parent.
 //
 // Since: 2.5
 func NewInnerWindow(title string, content fyne.CanvasObject) *InnerWindow {
-	w := &InnerWindow{title: title, content: content}
+	w := &InnerWindow{title: title, content: NewPadded(content)}
```

Comment:
I think by the "Principle of Least Surprise", inner windows should be unpadded by default, just like top-level windows. What about introducing a secondary constructor `NewPaddedInnerWindow` that sets it up with a padded container, and having this constructor set up a stack (ie unpadded) container?

Reply:
Top level windows are padded by default though, aren’t they?

Reply:
Oh wait, you're right. Disregard :)